Changes to setup DB using sqlite3 package instead of the process

darien-garcia opened the pull request:
I did this to avoid accessing the sqlite3 process in windows with carries more complexity to setup permissions

flacours commented:
Hi thank you for your good work. there are couple of things you need to do to help me review the test. first you need to adjust your requirement.txt file. It's incomplete you added new dependencies. 

second : the app is not starting. there is a crash on start up . in main.py `app=create_app()` . but create_app doesn't exit.

3rd. i don't know which version of python you are using. can you adjust the README.md ?

darien-garcia commented:
first: I've added the missing dependencies. Let me know if there's anything else missing
second: I added a couple of commits to change that
3rd: I'm using python 3.4.4

flacours commented:
hi, when i start `python main.py initdb` i get the following error

'SQLAlchemy' object has no attribute 'create_db_connection'

flacours commented:
also about the requirement I've added the following
Flask-Migrate
Flask-SQLAlchemy

and i have removed functools because it's standard in the python libraries.

flacours commented:
maybe you are using a different requirements ?

darien-garcia commented:
I've added back the sqlite3 database connection and creation. Also you'll need to create a migration and upgrade the database after you create it to add the 'completed' column to match the SQLAlchemy model
